flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/classification/SVM.scala

```diff
@@ -382,7 +402,11 @@ object SVM{
 
             // calculate the new weight vector by adding the weight vector updates to the weight
             // vector value
-            weights.union(weightedDeltaWs).reduce { _ + _ }
+            val newWeights = weights.union(weightedDeltaWs).reduce { _ + _ }
+            if(convergenceDefined){
+              converger.get.construct(input,weights,newWeights)
```

Comment:
@tillrohrmann , could you take a look here? I am unable to work out the problem here.